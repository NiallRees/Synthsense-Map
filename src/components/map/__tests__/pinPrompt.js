import PinPrompt from '../pinPrompt';
import React from 'react';
var enzyme = require('enzyme');

describe('Test PinPrompt', () => {
  it('Renders', () => {
    const component = enzyme.shallow(<PinPrompt pinPrompt={{enabled: true, latitude: 0, longitude: 0}}/>);
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  })
})