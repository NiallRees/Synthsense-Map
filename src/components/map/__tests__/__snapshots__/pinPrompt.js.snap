// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PinPrompt Renders 1`] = `ShallowWrapper {}`;
